pkgs/top-level/python-packages.nix

```diff
@@ -4726,6 +4726,8 @@ in {
 
   ndtypes = callPackage ../development/python-modules/ndtypes { };
 
+  nengo-gui = callPackage ../development/python-modules/nengo-gui { };
```

Comment:
This should be moved out of pythonPackages.

Reply:
moved where to ?

Reply:
top-level packages because it is not a python library but an end user application, right?

Reply:
it's a front end to a python library with a built-in editor, would applications/science/machine-learning or applications/editors be better suited for that ?

Reply:
> applications/editors

that would work for me.